sale_manual_delivery/hook.py

```diff
@@ -0,0 +1,64 @@
+def pre_init_hook(cr):
+    cr.execute(
+        """
+        ALTER TABLE sale_order_line ADD COLUMN IF NOT EXISTS qty_procured numeric;
+        COMMENT ON COLUMN sale_order_line.qty_procured IS 'Quantity Procured';
```

Comment:
Naïve question why to add comment on postgres field ?

Reply:
In this case I do it like this, to reproduce the native behavior of odoo when creating a new column